### PR TITLE
json5: add page

### PR DESCRIPTION
**Pull request by pxgamer**
----

## Basic Check

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited.

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines.

## References

Main page: http://json5.org
GitHub repository: https://github.com/json5/json5

### CODE REVIEW

File: pages/common/json5.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# json5
+
+> A command-line tool for transpiling JSON5 files to JSON.
```
Comment (agnivade) [inline pages/common/json5.md]: Any reason you are using the word "transpile" ? You are just converting files from one format to another. It's not even source code.

File: pages/common/json5.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+
+- Transpile a JSON5 file to a JSON file of the same name:
+
+`json5 -c {{path/to/file}}`
```
Comment (agnivade) [inline pages/common/json5.md]: Hmm .. I don't see the `-c` option anywhere here - https://github.com/json5/json5#cli. What am I missing ?

File: pages/common/json5.md (hunk 1 above)
Comment (pxgamer) [reply pages/common/json5.md]: I guess it's more just converting. :+1: I'll change that.

File: pages/common/json5.md (hunk 2 above)
Comment (pxgamer) [reply pages/common/json5.md]: Not sure why `-c` isn't listed as that's one of the main options. :thinking: Not sure why I missed the other available commands, I'll add them too.

Comment (pxgamer) [reply pages/common/json5.md]: Turns out my version was slightly out of date. I'll redo the page. _sigh_

**Comment by pxgamer** [comment]
Closing to redo with the correct parameters on the latest version.

**Comment by pxgamer** [comment]
Reopened as I've now updated it.

File: pages/common/json5.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
+# json5
+
+> A command-line tool for converting JSON5 files to JSON.
+
+- Convert JSON5 STDIN to JSON STDOUT:
+
+`{{input}} | json5`
```
Comment (agnivade) [inline pages/common/json5.md]: echo "{{input}}" ?

File: pages/common/json5.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
+# json5
+
+> A command-line tool for converting JSON5 files to JSON.
+
+- Convert JSON5 STDIN to JSON STDOUT:
```
Comment (agnivade) [inline pages/common/json5.md]: I think you can make stdin and stdout to lowercase.

File: pages/common/json5.md (hunk 3 above)
Comment (pxgamer) [reply pages/common/json5.md]: Sounds better 👍

**Pull request merged**